src/shared/components/Header/Navigation/styles.ts

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+
+export const Navigation = styled.nav`
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  ul {
+    display: flex;
```

Comment:
let's try to keep our styles as clean as possible and use bootstrap classes instead if possible, 

for example add `className="d-flex align-items-center"`